swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
+
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    
+
+# Wave 02
+    def __init__(self, category="", condition=0, id=0):
+        if id is 0:
```

Comment:
`id` will typically/always be a number. In Python, it's better to use`==` rather than `is` when comparing numbers.